CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+### Contributing
+
+To contribute to the codebase, you will need to fork the repository.
+
+The following steps will get you setup to contribute changes to this repo:
+
+- Fork the repo and create your branch from main.
+- Install depencencies and build.
+
+We use npm to manage installation of dependencies and running various scripts. To get everything installed, make sure you have npm and run npm install from the root of the repository.
+
+#### Reporting new issues
+
+When opening a new issue, always make sure to fill out the issue template.
+
+- One issue, one bug: Please report a single bug per issue.
+- Provide reproduction steps: List all the steps necessary to reproduce the issue. The person reading your bug report should be able to follow these steps to reproduce your issue with minimal effort.
+
+#### Development
```

Comment:
Are you planning to fill this section?

Reply:
Actually, development is like the main title of the following titles here.

Reply:
oh sorry, makes sense